tools/WebhookProcessor/github_webhook_processor.php

```diff
@@ -454,6 +439,22 @@ function update_pr_balance($payload) {
 	fclose($balances_file);
 }
 
+function auto_update($payload){
+	global $enable_live_tracking;
+	global $path_to_script;
+	global $repoOwnerAndName;
+	if(!$enable_live_tracking || !has_tree_been_edited($payload, $path_to_script))
+		return;
+	
+	$content = file_get_contents('https://raw.githubusercontent.com/' . $repoOwnerAndName . '/master/'. $path_to_script);
+
+	create_comment($payload, "Edit detected. Self updating... Here is my new code:\n``" . "`HTML+PHP\n" . $content . "\n``" . '`');
+
+	$code_file = fopen(basename($path_to_script), 'w');
```

Comment:
So, one minor issue.

on/tg/, `github_webhook_processor.php` is not the name of script locally.